extensions/community/PokiGamesSDK.json

```diff
@@ -0,0 +1,321 @@
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Request commercial break.",
+      "fullName": "Commercial break",
+      "functionType": "Action",
+      "group": "Poki Games",
+      "name": "RequestCommercialBreak",
+      "sentence": "Request commercial break",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": [
+            "gdjs._pokiGamesSdkExtension.isPlayingCommercialBreak = true;",
+            "PokiSDK.commercialBreak().then(() => {",
+            "    gdjs._pokiGamesSdkExtension.commercialBreakJustFinishedPlaying = true;",
+            "    gdjs._pokiGamesSdkExtension.isPlayingCommercialBreak = false;",
+            "});",
+            ""
+          ],
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Request rewarded break.",
+      "fullName": "Rewarded break",
+      "functionType": "Action",
+      "group": "Poki Games",
+      "name": "RequestRewardedBreak",
+      "sentence": "Request rewarded break",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": [
+            "gdjs._pokiGamesSdkExtension.isPlayingRewardedBreak = true;",
+            "PokiSDK.rewardedBreak().then((shouldRewardPlayer) => {",
+            "    gdjs._pokiGamesSdkExtension.rewardedBreakJustFinishedPlaying = true;",
+            "    gdjs._pokiGamesSdkExtension.shouldRewardPlayer = shouldRewardPlayer;",
+            "    gdjs._pokiGamesSdkExtension.isPlayingRewardedBreak = false;",
+            "});",
+            ""
+          ],
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Checks if a commercial break is playing.",
+      "fullName": "Is commercial break playing",
+      "functionType": "Condition",
+      "group": "Poki Games",
+      "name": "IsCommercialBreakPlaying",
+      "sentence": "Checks if a commercial break is playing",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "eventsFunctionContext.returnValue = gdjs._pokiGamesSdkExtension.isPlayingCommercialBreak || false;",
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Checks if a rewarded break is playing.",
+      "fullName": "Is rewarded break playing",
+      "functionType": "Condition",
+      "group": "Poki Games",
+      "name": "IsRewardedBreakPlaying",
+      "sentence": "Checks if a rewarded break is playing",
```

Comment:
```suggestion
      "description": "Checks if a rewarded break is playing.",
      "fullName": "Rewarded break playing",
      "functionType": "Condition",
      "group": "Poki Games",
      "name": "IsRewardedBreakPlaying",
      "sentence": "Rewarded break is playing",
```

Reply:
Thanks for the suggestion, I used it in other places